components/EmptyTokensText.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Subheading } from 'react-native-paper'
+
+import theme from '../defaultTheme'
+
+import Spacer from './Spacer'
+
+const UI_STRINGS = {
+  default: {
+    heading: `You don't have a secret set up yet.`,
+    description:
+      'Add one by scanning or uploading a QR code, or even enter the details manually.',
+  },
```

Comment:
why the nesting?

Reply:
This was a relic from the previous implementation. No need for it anymore, you're right. I've removed the nesting.